lib/tls/session.toit

```diff
@@ -121,6 +121,7 @@ class Session:
   certificate/Certificate?
   root-certificates/List
   handshake-timeout/Duration
+  skip-certificate-verification/bool
```

Comment:
I think this should be validation, not verification.

See https://datatracker.ietf.org/doc/html/rfc4949 under "validate vs. verify" on page 330.

Reply:
done.